src/components/Header/hook.ts

```diff
@@ -0,0 +1,12 @@
+import { useLocation } from 'react-router-dom';
+
+import { MENU_PATH } from 'src/features/Router/path';
+
+export const useHeader = () => {
+  const location = useLocation();
```

Comment:
you have props for this reason, location is redundant

Reply:
done

Reply:
Done